### PR TITLE
Also show cartesian areas/perimeters in identify results

### PR DESCRIPTION
**Pull request by nyalldawson**
Because users are still getting confused with the difference
between the cartesian areas and ellipsoidal areas, show both
in the identify results dock.

The reasoning here is:

- if a user understands this concept, they will know the correct
one to use, and its good for us to inform them of the difference
here. Plus, it means they immediately see if the ellipsoid
setting is correct and the difference it is making for the
area/length calculation.

- if a user has no idea what the difference is, then we should
make them aware that there's (at least!) two different possible
measurement values. They can then either research what these mean and make
the right choice (and become better informed GIS practitioners),
OR pick a random one - which really is no different then the
previous situation, because an uniformed user is just as likely
to be working in an unsuitable projection with a poor ellipsoid
choice.

In short, we don't try to guess the right choice for users
and instead give them all the information and let them make the
call which value to use.

Inspired by https://gis.stackexchange.com/questions/272213/why-area-gave-me-a-wrong-value-qgis-2-99-solved?noredirect=1#comment433911_272213 .

I'm keen for feedback here.

### CODE REVIEW

**Comment by lbartoletti** [comment]
Very good idea. Thanks.

**Comment by jguelat** [comment]
As a user I was a little bit confused since I didn't notice that the default changed. If I understood correctly QGIS is now automatically using the ellipsoid associated to the project CRS (in QGIS 2.18 I had OTF transformation off and then cartesian/planimetric was automatically selected).

I agree that using an ellipsoidal area is the right decision if people have data in a geographic CRS (and this will avoid a lot of mistakes). However I have the feeling that most people using a projected CRS are more interested in (and used to) the cartesian area. I see 3 possibilities:

1. Leave it as it is. You need to change each project to planimetric if you don't want the ellipsoidal area.
2. Add a QGIS-wide setting allowing automatic switching to planimetric if the CRS is projected.
3. Switch the ellipsoid automatically to planimetric if the CRS is projected.

And going back to the original question in this PR: I also think it would be nice to see the 2 values in the identify results.
